bot/exts/info/information.py

```diff
@@ -41,6 +42,17 @@ def get_channel_type_counts(guild: Guild) -> DefaultDict[str, int]:
 
         return channel_counter
 
+    @staticmethod
+    def join_role_stats(role_ids: List[int], name: str, guild: Guild) -> Dict[str, int]:
```

Comment:
```suggestion
    def join_role_stats(role_ids: list[int], name: str, guild: Guild) -> dict[str, int]:
```
Like I mentioned we can use the builtin generics now, you can also remove the List import with this

Reply:
Most of the code still uses the typing module, I thought I should change it. But I'll change it then.